test/ios/mocked_data/test_get_bgp_neighbors/572/show_bgp_ipv4_unicast_neighbors.txt

```diff
@@ -1,6 +1,7 @@
 BGP neighbor is 192.168.0.2,  remote AS 65001, internal link
  Inherits from template iBGP for session parameters
   BGP version 4, remote router ID 0.0.0.0
+  BGP state = Idle
```

Comment:
I don't think this is correct. We shouldn't manipulate mocked data to have tests pass. If this line wasn't here is because this device doesn't report it.